### PR TITLE
Fix various issues pointed out by CodeQL

### PR DESCRIPTION
**Pull request by LordAro**
I think that's most of them. PRing so I can see if there's anything left.

I've marked everything in bottle.py as "won't fix". I'd suggest that I'd report them upstream, but nah.

### CODE REVIEW

File: webtranslate/__main__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 from . import bottle
 from . import main
-from .click import click_additional_options
+from .click_extras import click_additional_options
```
Comment (TrueBrain) [inline webtranslate/__main__.py:6]: On the other repositories I marked this error as "won't fix", as CodeQL is wrong here. `.click` and `click` in Python3 are completely different packages, so the error itself is not correct. One can, however, debate that the naming can be confusing. But if we go this route, we might as well fix it for all the other repositories too :)

File: scripts/lang_sync [hunk 2]
```diff
@@ -366,11 +366,7 @@ def post_form_upload(url, form_data, form_fentry, form_fname, fdata):
 
     # Perform the request, and return the read data.
     with urllib.request.urlopen(req) as resp:
-        if resp.status != 200:
-            return False
-
-        data = resp.read()
```
Comment (TrueBrain) [inline scripts/lang_sync:372]: not doing this `read()` can have side-effects. You sure this is okay? (mind you, `eints` uses `urllib`, not `urllib3` ...)

File: webtranslate/__main__.py (hunk 1 above)
Comment (LordAro) [reply webtranslate/__main__.py:6]: Mm, that's probably the better solution

File: scripts/lang_sync (hunk 2 above)
Comment (LordAro) [reply scripts/lang_sync:372]: Can it? We clearly don't care about the content of the response, and have already used the status...

Comment (TrueBrain) [reply scripts/lang_sync:372]: I honestly don't know, that is why I ask :D (it was an honest question). I remember a lot of quirks with `urllib`. Especially when pooling is used. So basically what I tried to ask: did you test if this is still working correctly? :D

File: scripts/lang_sync [hunk 3]
```diff
@@ -1269,19 +1267,19 @@ def run():
 
     setup_auth()
 
-    # Open 'credits' file.
-    if user_cfg.credits_file is None:
+    try:
         credits_handle = None
```
Comment (TrueBrain) [inline scripts/lang_sync:1271]: Please move the `credits_handle = None` out of the try block.

**Review by TrueBrain — CHANGES_REQUESTED** [review]

File: scripts/lang_sync (hunk 2 above)
Comment (TrueBrain) [reply scripts/lang_sync:372]: On line 320 something similar happens. Possibly use what they did there?

**Review by TrueBrain — APPROVED** [review]
LGTM

**Pull request merged**